tests/conftest.py

```diff
@@ -198,6 +200,50 @@ def _make_client(auth: Auth) -> RemoteSettingsClient:
     return _make_client
 
 
+@pytest.fixture(scope="session")
+def setup_client(setup_auth, make_client) -> ClientFactory:
```

Comment:
It seems like a lot of place where we use `ClientFactory` as a type, we want to use `RemoteSettingsClient`. Only `make_client` returns a `ClientFactory`, but the factory returns a `RemoteSettingsClient`.

Reply:
Yup, making that change now.